source/text/Format.cpp

```diff
@@ -293,20 +293,23 @@ string Format::Credits(int64_t value)
 	int64_t absolute = abs(value);
 
 	// Handle numbers bigger than a million.
-	static const vector<char> SUFFIX = {'T', 'B', 'M'};
-	static const vector<int64_t> THRESHOLD = {1000000000000ll, 1000000000ll, 1000000ll};
-	for(size_t i = 0; i < SUFFIX.size(); ++i)
-		if(absolute > THRESHOLD[i])
+	static constexpr array<pair<int64_t, char>, 3> THRESHOLD_SUFFIX = {{
+		{1000000000000ll, 'T'},
+		{1000000000ll, 'B'},
+		{1000000ll, 'M'}
+	}};
```

Comment:
why are the outer braces duplicated?

Reply:
> why are the outer braces duplicated?

https://stackoverflow.com/questions/14178264/c11-correct-stdarray-initialization

Reply:
an example here
https://en.cppreference.com/w/cpp/container/array
says they aren't needed in C++14 or after `=`

Reply:
I don't know why, but it won't compile without the extra braces on my system (regardless of clang or gcc compiler).
I can push it and see if it builds in CI.

Reply:
Doesn't work. I'm going to guess the reason it doesn't is that `std::pair` isn't a primitive.

Reply:
So the answer is yes; they are needed.